synapse/lib/lmdblayer.py

```diff
@@ -94,6 +100,29 @@ class LmdbLayer(s_layer.Layer):
             'range': self._rowsByRange,
         }
 
+    async def setNodeData(self, buid, name, item):
+        # TODO more str->utf8 caching
+        utf8 = name.encode('utf8')
+        self.dataslab.put(utf8 + b'\x00' + buid, b'\x00', db=self.databyname)
+        self.dataslab.put(buid + utf8, s_msgpack.en(item), db=self.databybuid)
+
+    async def getNodeData(self, buid, name, defv=None):
+        utf8 = name.encode('utf8')
+        byts = self.dataslab.get(buid + utf8, db=self.databybuid)
```

Comment:
since nothing else is using uf8 in this function we can just do `byts = self.dataslab.get(buid + name.encode(), db=self.databybuid)`